README.md

```diff
@@ -33,6 +33,7 @@ _These assume you have `egghead-systems` running locally_
 - **Package management:** Yarn
 - **Scripts:** Yarn
 - **JS Flavor:** ES2016+ _(default `create-react-app`)_
+- **JS Utils:** Lodash
```

Comment:
Good point; we should try it out. I'm new to Rx but would like to learn more. I wonder if using Rx in place of Lodash for sync stuff would have any negative consequences since it creates observables that you don't really need since you are just doing stuff once / sync?